### PR TITLE
Add djLint lint/check when running lint-python

### PR DESCRIPTION
**Pull request by alexandreNoens**

### CODE REVIEW

**Review by glepretre — APPROVED** [review]

File: base.Makefile [hunk 1]
```diff
@@ -324,6 +324,8 @@ clea%: least-specific-clean ## clean: Clean all built assets
 lint-pytho%: ## lint-python: Lint python source
 	$(LOG)
 	$(RUFF) check $(LINTED_PYTHON_DIRS)
+	$(DJLINT) $(TEMPLATE_DIRS) --lint
```
Comment (gentooboontoo) [inline base.Makefile:328]: Please add a `TEMPLATE_DIRS` variable default value into `## Python env` block in this file. Something like: 

`TEMPLATE_DIRS ?= $(PYTHON_SRCDIR)/templates`

**Review by gentooboontoo — CHANGES_REQUESTED** [review]

File: base.Makefile (hunk 1 above)
Comment (gentooboontoo) [reply base.Makefile:328]: Please check that it does not break.

Comment (alexandreNoens) [reply base.Makefile:328]: Done :+1:

**Review by gentooboontoo — APPROVED** [review]
LGTM :+1: You can squash both commits.

**Review by glepretre — COMMENTED** [review]
Mob : Ship it! :rocket:

**Pull request merged**